pkg/deploy/internal/classic/validation.go

```diff
@@ -42,7 +42,7 @@ func (v *Validator) Validate(c config.Config) error {
 	}
 
 	a, ok := c.Type.(config.ClassicApiType)
-	if !ok || apis[a.Api].NonUniqueName {
+	if !ok || apis[a.Api].NonUniqueName || apis[a.Api].HasParent() {
```

Comment:
While this indicates that they _could_ have a different parent is it guaranteed? What if the scope is the same and the name is also the same?

Reply:
true, and good catch :) - i should have mentioned that we already have a ticket that's is targeting this particular issue